Applications/mnist/Tensorflow/Training_Keras.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2020 Jijoong Moon <jijoong.moon@samsung.com>
+#
+# SPDX-License-Identifier: Apache-2.0-only
+#
+# @file	mnist_Keras.py
+# @date	13 July 2020
+# @brief	This is Simple Classification Example using Keras
+# @see		https://github.com/nnstreamer/nntrainer
+# @author	Jijoong Moon <jijoong.moon@samsung.com>
+# @bug		No known bugs except for NYI items
+#
+# mnist example
+# inputlayer -> conv2d (5x5) 6 filters -> pooling2d 2x2 (valid) -> conv2d (5x5) 12 filters ->
+#               pooling2d 2x2 (valid) -> flatten -> fully connected layer -> 10 class
+#
+
+import random
+import struct
+import os
+import tensorflow as tf
+import numpy as np
+from tensorflow.keras.layers import Input, Dense, Conv2D, Flatten, AveragePooling2D
+from tensorflow.keras import models, layers, optimizers
+from tensorflow.keras.callbacks import Callback
+
```

Comment:
lets take deterministic code making sections from input_gen.py
or may be add them to a separate file and use it everywhere to be able to reproduce values from tensorflow.

Reply:
done. Thanks

Reply:
Add `os.environ['TF_CUDNN_DETERMINISTIC'] = '1'` before import tf

Add below below here.
```
# Fix the seeds across frameworks
SEED = ....
tf.compat.v1.reset_default_graph()
random.seed(SEED)
tf.compat.v1.set_random_seed(SEED)
np.random.seed(SEED)
```

Reply:
done